clusters/l3-sqnc/monitoring/jaeger/values.yaml

```diff
@@ -0,0 +1,88 @@
+provisionDataStore:
+  cassandra: false
+  elasticsearch: true
+storage:
+  type: elasticsearch
+elasticsearch:
+  master:
+    replicaCount: 3
+    persistence:
+      storageClass: managed-csi-premium
+      size: 50Gi
+  data:
+    replicaCount: 3
+    persistence:
+      storageClass: managed-csi-premium
+      size: 80Gi
```

Comment:
Do we really need so much redundency here? This seems overkill since we do not even have replicas of our application databases

Reply:
You need a minimum or two for each of these, there can be quorum issues with lesser numbers.